pom.xml

```diff
@@ -51,7 +51,7 @@
         <apache.httpclient.version>4.5.3</apache.httpclient.version>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
         <jersey.version>2.28</jersey.version>
-        <jetty.version>9.4.20.v20190813</jetty.version>
+        <jetty.version>9.4.23.v20191118</jetty.version>
```

Comment:
is there a reason we're bumping?

Reply:
so we don't have to write our own OIDC authenticator

Reply:
This was the earliest version which includes it so I'm hopeful it has a lesser chance of breaking things unexpectedly.